### ISSUE TITLE
aria-roledescription should not be allowed on role generic

### ISSUE THREAD

**Issue by scottaohara** [w3c/aria#1140]
On the call today we decided it was an oversight that `role=generic` will now be allowed to have `aria-roledescription`.

`aria-roledescription` should be prohibited from being used on `role=generic`

**Comment by scottaohara** [comment]
also, do we still need this note in `generic`'s definition?

>A triage of global properties is pending. This will reduce the number of inherited states and properties in the characteristics table below. For details, see issue #999`

the issue is closed, so saying the triage of properties is pending isn't accurate

**Comment by carmacleod** [comment]
I think we can remove the note. We did remove 4 attributes from the globals, and made aria-label[ledby] prohibited on generic, and now aria-[braille]roledescription as well, so that's 8 fewer globals than when the note was written. Sure, we may remove more globals, or prohibit other attributes on `generic`, but the note is not particularly useful as a placeholder for possible further triage.

**Comment by scottaohara** [comment]
just pushed a change to remove the note.

**Comment by LaurenceRLewis** [comment]
> aria-label[ledby] prohibited on generic

DIVs and SPANs but what other elements are considered ‘generic’? 

Thank you

**Comment by carmacleod** [comment]
@LaurenceRLewis 
The plans for `generic` are listed on the [role parity page](https://github.com/w3c/aria/wiki/Plans-regarding-role-parity#parity-achieved-in-aria--12). Specifically, the `generic` elements are/will be:
> - generic (with no additional attribute): div, span, cite, footer and header (not scoped to body), form and section (without accessible name), tfoot, thead, details
>
> - generic (an additional attribute may be created in 1.3): abbr, address, b, i, kbd, mark, pre, q, s, samp, small, u, var
